cmd/operator-sdk/generate/cmd.go

```diff
@@ -16,15 +16,29 @@ package generate
 
 import (
 	"github.com/spf13/cobra"
+
+	"github.com/operator-framework/operator-sdk/cmd/operator-sdk/generate/bundle"
 )
 
-func NewCmd() *cobra.Command {
-	cmd := &cobra.Command{
+func newCmd() *cobra.Command {
+	return &cobra.Command{
 		Use:   "generate <generator>",
 		Short: "Invokes a specific generator",
 		Long: `The 'operator-sdk generate' command invokes a specific generator to generate
-code or manifests on disk.`,
+code or manifests.`,
 	}
+}
+
+func NewCmd() *cobra.Command {
+	cmd := newCmd()
+	cmd.AddCommand(
+		bundle.NewCmd(),
+	)
+	return cmd
+}
+
+func NewCmdLegacy() *cobra.Command {
```

Comment:
```suggestion
// todo: remove when the legacy layout is no longer supported
--
// Deprecated: Used just for the legacy layout
// --
func NewCmdLegacy() *cobra.Command {
```

Reply:
Meta note: do not set `Deprecated` on exported functions still in use. Doing so will break the linter.